jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/actions/DeployServerlessApplicationAction.kt

```diff
@@ -142,6 +144,8 @@ class DeployServerlessApplicationAction : AnAction(
                     project
                 )
                 SamTelemetry.deploy(project, Result.Succeeded)
+                // We do not know if it is a create or not, so refresh every time
+                project.refreshAwsTree(CloudFormationResources.LIST_STACKS)
```

Comment:
Since CFN can touch anything and everything, should this be a full refresh?

Reply:
Yep, this seems like a reasonable place to do that, changed